pkgs/development/python-modules/aeidon/default.nix

```diff
@@ -13,15 +14,17 @@ buildPythonPackage rec {
   pname = "aeidon";
   version = "1.15";
 
+  pyproject = true;
+
   src = fetchPypi {
     pname = "aeidon";
     inherit version;
-    sha256 = "sha256-qGpGraRZFVaW1Jys24qvfPo5WDg7Q/fhvm44JH8ulVw=";
+    hash = "sha256-qGpGraRZFVaW1Jys24qvfPo5WDg7Q/fhvm44JH8ulVw=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
+    setuptools
     gettext
```

Comment:
That's not the build-system.

Reply:
Though it's required at build time. Should I move it into nativeBuildInputs then?